ci-operator/step-registry/openshift/e2e/gcp/libvirt/conf/OWNERS

```diff
@@ -0,0 +1,7 @@
+approvers:
```

Comment:
nit: might be easier to maintain if you use a symlink here; compare #12832.

Reply:
deduped ci-op/step-registry/openshfit/e2e/gcp/libvirt, thanks! did not know of this script b4 : )